src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HearingsHelper.java

```diff
@@ -87,4 +91,32 @@ public static void updatePostponedDate(CaseData caseData) {
         }
 
     }
+
+    public static List<String> hearingTimeValidation(CaseData caseData) {
+        List<String> errors = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(caseData.getHearingCollection())) {
+            for (HearingTypeItem hearingTypeItem : caseData.getHearingCollection()) {
+                if (CollectionUtils.isNotEmpty(hearingTypeItem.getValue().getHearingDateCollection())) {
+                    for (DateListedTypeItem dateListedTypeItem :
+                            hearingTypeItem.getValue().getHearingDateCollection()) {
+                        var dateListedType = dateListedTypeItem.getValue();
+                        if (HEARING_STATUS_HEARD.equals(dateListedType.getHearingStatus())) {
+                            checkStartFinishTimes(errors, dateListedType,
+                                    hearingTypeItem.getValue().getHearingNumber());
+                        }
+                    }
+                }
+            }
+        }
+        return errors;
+    }
+
+    private static void checkStartFinishTimes(List<String> errors, DateListedType dateListedType,
+                                              String hearingNumber) {
+        var startDate = LocalDateTime.parse(dateListedType.getHearingTimingStart());
+        var finishTime = LocalDateTime.parse(dateListedType.getHearingTimingFinish());
+        if (startDate.isAfter(finishTime) || startDate.isEqual(finishTime)) {
```

Comment:
if (!startDate.isBefore(finishTime)) {
maybe?

Reply:
changed to if (finishTime.isBefore(startTime)
in terms of the ticket, i think it makes more sense if it's in that order